### DISCUSSION TITLE
npm run dist error mediacms-scripts: not found

### DISCUSSION THREAD

**Discussion by egiaggata** [mediacms-io/mediacms#914] [Q&A] [answered]
please help, when i use npm run dist error mediacms-scripts: not found in the folder frontend

**Comment by mgogoulos** [comment, marked as answer]
How do you run it? Check how it can be run through docker-compose - https://github.com/mediacms-io/mediacms/blob/main/docs/dev_exp.md

**Reply by egiaggata** [reply]
Thanks mgogoulos, I have succeeded.

I'm building a video sharing portal with a website address https://netters.zone
